lib/public_key/include/public_key.hrl

```diff
@@ -78,6 +78,8 @@
 	  point
 	 }).
 
+-record(cert, {der, otp}).
+
 -define(unspecified, 0).
 -define(keyCompromise, 1).
 -define(cACompromise, 2).
```

Comment:
Why cACompromise and not caCompromise?

Reply:
These are enumeration names from an RFC and defined in an PKIX ASN-1 spec, and also not part of my PR!